### PR TITLE
Add github action for openstack-ansibleee-runner build

### PR DESCRIPTION
**Pull request by bshewale**

### CODE REVIEW

File: .github/workflows/openstack-ansibleee-runner.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    steps:
+      - name: Check secrets are set
+        id: have-secrets
+        if: "${{ env.imagenamespace != '' }}"
+        run: echo "::set-output name=ok::true"
+    outputs:
+      have-secrets: ${{ steps.have-secrets.outputs.ok }}
+
+  build-openstack-ansibleee-runner:
+    name: Build openstack-ansibleee-operator image using buildah
```
Comment (rebtoor) [inline .github/workflows/openstack-ansibleee-runner.yaml:34]: `openstack-ansibleee-runner` image

File: .github/workflows/openstack-ansibleee-runner.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+jobs:
+
+  check-secrets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check secrets are set
+        id: have-secrets
+        if: "${{ env.imagenamespace != '' }}"
+        run: echo "::set-output name=ok::true"
```
Comment (rebtoor) [inline .github/workflows/openstack-ansibleee-runner.yaml:29]: Github has deprecated `::set-output`, read [here](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

You can use `run: echo "ok=true" >> $GITHUB_OUTPUT`

**Review by rebtoor — CHANGES_REQUESTED** [review]
Thanks a lot for this PR! I left a couple of comments for few minor tweaks.

File: .github/workflows/openstack-ansibleee-runner.yaml (hunk 1 above)
Comment (bshewale) [reply .github/workflows/openstack-ansibleee-runner.yaml]: Done

**Review by rebtoor — APPROVED** [review]

**Pull request merged**